feat: Apply intel-nuc templates rebrand

mtruj013 opened the pull request:
## Done

- Applied rebrand as per [design](https://www.figma.com/design/POgmRw4Rpq1hlgaD9DN8mS/ubuntu.com%2Fappliance?node-id=100-3425&p=f&t=vSzuQS0Cl4pWmAG1-0)

## QA

- View the site locally in your web browser at: 
    - https://ubuntu-com-14897.demos.haus/appliance/openhab/intel-nuc
    - https://ubuntu-com-14897.demos.haus/appliance/lxd/intel-nuc
    - https://ubuntu-com-14897.demos.haus/appliance/adgaurd/intel-nuc
    - https://ubuntu-com-14897.demos.haus/appliance/plex/intel-nuc
    - https://ubuntu-com-14897.demos.haus/appliance/mosquitto/intel-nuc
    - Be sure to test on mobile, tablet and desktop screen sizes
- Click through OS tabs and check against design and live site

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-16589

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-14897.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/7323)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

Sophie-32 commented:
The adguard demo wouldn't open, checked the others and left comments in the Figma since they follow the template and all meaningful copy is from live site

Sophie-32 commented:
- [ ] clicking tab interaction > it scrolls past the tab labels. Tab labels should stay visible when clicked.
- [ ] clicking away from 'how to verify your download' tool tip doesn't collapse it. Should collapse when use clicks away and/or clicks the linked text again

kim-isaac commented:
Hi @mtruj013 Thanks for your work! I have some comments below:
you can also check all the comments on the [figma file](https://www.figma.com/design/POgmRw4Rpq1hlgaD9DN8mS?node-id=100-3425#1187902374)

For hero section,
![Screenshot 2025-03-25 at 11 56 11](https://github.com/user-attachments/assets/197a2f17-e289-4a37-895b-0e197de3f614)

- [ ] Could you change the logo size to be 36px height?
- [ ] for the text link, icon should also follow the link color
- [ ] Could you place H2 ‘installation instructions’ heading here with rule?

After the tab,
![Screenshot 2025-03-25 at 11 57 43](https://github.com/user-attachments/assets/a3ac3d82-6cbf-45e5-8adc-d4a53ca8f9fe)

- [ ] After H2 ‘installation instructions’ is added, these should be tagged as h3s

![Screenshot 2025-03-25 at 11 58 27](https://github.com/user-attachments/assets/9ca20102-9e3a-4ce5-bf8a-2a9211fc6ef3)

- [ ] stepped title should be H4
- [ ] between how to install and 1. install OpenSSH: needs shallow margin(24px) and rule should be removed
- [ ] for the stepped item: please use [[p-stepped-list__title](https://vanillaframework.io/docs/patterns/lists#vertical-stepped)] style

For this kind of sections that code follow after text line break,
![Screenshot 2025-03-25 at 12 00 58](https://github.com/user-attachments/assets/c4ce3fa4-4da7-429a-a600-800cb7a9f52c)

for the code style - in the current live version, inline/block code style are mixed in these kind of sections. For consistency, I’ve changed all to block style in the mockup. Would it be possible to update them to block style as well? If you think inline style is more appropriate in this context, please let me know!

For 4th-6th items,
![Screenshot 2025-03-25 at 12 00 58](https://github.com/user-attachments/assets/04cd70d9-7712-4b42-b9e8-52acac1e760b)

- [ ]  please also use [[p-stepped-list__title](https://vanillaframework.io/docs/patterns/lists#vertical-stepped)] style
- [ ] Between the rule and contents, please add rule's bottom padding + text margin
- [ ] inside of the stepped item: please add text bottom margin between the text and code block

For 7th item,
![Screenshot 2025-03-25 at 12 05 49](https://github.com/user-attachments/assets/47f2ca15-52cb-4726-9315-6c0f36df3cef)

- [ ] please add rule's bottom padding
- [ ] seems shallow bottom padding(24px) is duplicated 3 times - could you remove all and add regular spacing? (64px for big screen / 32px for small and medium screen)

For small-medium dimensions,

![Screenshot 2025-03-25 at 12 08 47](https://github.com/user-attachments/assets/7808ba49-1337-4127-bd1c-13e2ce1825ee)

- [ ] for 4th to 7th items: Just like in the other sections, the content should be indented to match the title's text area.

That's all from my side. If you have any questions or thoughts regarding this, please let me know!
Thanks again

mtruj013 commented:
@Sophie-32 @kim-isaac thanks both! I've addressed most of the comments if y'all could take another look? Few comments/qs as well:

UX:
> clicking away from 'how to verify your download' tool tip doesn't collapse it. Should collapse when use clicks away and/or clicks the linked text again

The tooltip collapses on click, is this what you meant? 

Visual:
> for the text link, icon should also follow the link color

This is not currently possible on vanilla. There are some [theme options](https://vanillaframework.io/docs/base/themes), but none that match that blue link color 

>  Between the rule and contents, please add rule's bottom padding + text margin

Sorry, I don't understand what you're saying here and in similar comments. Let me know if I implemented what you meant!

Sophie-32 commented:
> @Sophie-32 @kim-isaac thanks both! I've addressed most of the comments if y'all could take another look? Few comments/qs as well:
> 
> UX:
> 
> > clicking away from 'how to verify your download' tool tip doesn't collapse it. Should collapse when use clicks away and/or clicks the linked text again
> 
> The tooltip collapses on click, is this what you meant?

Yes, tooltip collapses on click. Thanks!

kim-isaac commented:
Hi @mtruj013 , thanks for handling everything so well! 

> Sorry, I don't understand what you're saying here and in similar comments. Let me know if I implemented what you meant!
Yes, the change you made is exactly what I was asking for. TY

I have a few comments remaining:

- [ ] I realized it's currently showing a 50-50 split on medium dimension. Would it be possible to align it as a single column, as shown in the design mockup?
- [ ] As I asked on MM, could you check if we can have indentation on small - medium dimension?
- [ ] Looks like the pages from LXD to Mosquitto were added, but the bottom section still shows the live version. Is that part still being worked on?

mtruj013 commented:
> Looks like the pages from LXD to Mosquitto were added, but the bottom section still shows the live version. Is that part still being worked on?

Yes, that's a separate template. This pr is already 12 files deep so I thought it better to split up the work

kim-isaac commented:
Looks great, thank you @mtruj013 !!

petesfrench commented:
@eliman11 For the content at the bottom of these pages [the copydoc](https://docs.google.com/document/d/1gb03YbX5ZQ_oO74M36b3ybCPwqq29d8Q17c6KaguW18/edit?tab=t.0#) only contains the content for [/appliance/openhab/intel-nuc](/appliance/openhab/intel-nuc). I think we should also include the sections from the other pages here.

petesfrench commented on templates/appliance/openhab/_next-steps.html:
```diff
@@ -1,30 +1,33 @@
-          Start using your {{ appliance.name }} Ubuntu Appliance
+          Start using your
+          <br />
+          {{ appliance.name }} Ubuntu Appliance
```
Is there a scenario where the appliance wouldn't be 'openhab'?
```suggestion
          openHAB Ubuntu Appliance
```
I think this has to always be 'openHAB'

petesfrench commented on templates/appliance/shared/_steps_ssh-keys_ubuntu.html:
```diff
@@ -10,15 +8,21 @@ <h4 class="p-stepped-list__title">
     <pre><code>ssh-keygen -t rsa</code></pre>
-    <ol>
-      <li>
+    <ol class="">
```
There are some empty classes here

petesfrench commented:
I was unable to switch tab with keyboard alone. I tried using `tab` and also the arrow keys.

petesfrench reviewed (approved):
LGTM
